boards/variscite/imx8mp_var_dart/imx8mp_var_dart_mimx8ml8_a53.dts

```diff
@@ -49,8 +49,7 @@
 			interrupt-names = "irq_0";
 			interrupt-parent = <&gic>;
 			clocks = <&ccm IMX_CCM_UART3_CLK 0x68 12>;
-			rdc = <(RDC_DOMAIN_PERM(A53_DOMAIN_ID, RDC_DOMAIN_PERM_RW)|\
-				RDC_DOMAIN_PERM(M7_DOMAIN_ID, RDC_DOMAIN_PERM_RW))>;
+			rdc = <(RDC_DOMAIN_PERM(A53_DOMAIN_ID, RDC_DOMAIN_PERM_RW) |\ RDC_DOMAIN_PERM(M7_DOMAIN_ID, RDC_DOMAIN_PERM_RW))>;
```

Comment:
stray `\` here

Reply:
Bug found :) thanks for spotting this :)